custom_metrics/almanac.js

```diff
@@ -307,5 +307,7 @@ return JSON.stringify({
     }
   })(),
   //  check if there is any picture tag containing an img tag
-  'has_picture_img': document.querySelectorAll('picture img').length > 0
+  'has_picture_img': document.querySelectorAll('picture img').length > 0,
+  // check if there are any source or img tags with sizes attribute
+  'has_image_sizes': document.querySelectorAll('source[sizes], img[sizes]').length > 0  
```

Comment:
It might be more versatile to return the length itself rather than the `> 0` condition. This enables followup queries like "median number of images with sizes per page" for example.

(Also applicable to `has_picture_img`. You may also want to rename this from "has" to "num" or similar.)